CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Main releases
 
+# 3.1.0
+* Add serviceUuids to `ScanResult`. Kudos to @xvrh.
```

Comment:
"serviceUuids" is not natural language and not formatted as `code`. Decide what you want it to looks and update text or formatting, please.